src/MOI_bounded_oracle.jl

```diff
@@ -303,6 +303,7 @@ end
 Check if problem is bounded and feasible, i.e. no contradicting constraints.
 """
 function check_feasibility(blmo::MathOptBLMO)
+    free_model(blmo)
```

Comment:
@dhendryc not sure if this is necessary with the changes from last Friday, feel free to remove

Reply:
I removed it. It's not necessary.